# This message type is for sending debug information
# to the debug node from the other components
# in the project

string node_name
string debug_message